### PR TITLE
Moving some more across

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/shape/getTags.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
+/*
 import Shape from './Shape.js';
 
 // This should take an op.
 
-export const getTags = Shape.registerMethod2(
+export const getTags = Shape.registerMethodX(
```
Comment (BarbourSmith) [inline api/shape/getTags.js:6]: What is registerMethodX?

**Review by BarbourSmith — APPROVED** [review]
Looks great to me :)

Sorry this one took me so long to read through

File: api/shape/getTags.js (hunk 1 above)
Comment (pentacular) [reply api/shape/getTags.js:6]: Ah, just something so that the commented out code didn't match registerMethod2 anymore -- I'll remove that.

**Comment by pentacular** [comment]
> Looks great to me :)
> 
> Sorry this one took me so long to read through

No problem :)

Thanks for taking a look.

**Pull request merged**